resource-managers/mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosCoarseGrainedSchedulerBackend.scala

```diff
@@ -422,6 +431,21 @@ private[spark] class MesosCoarseGrainedSchedulerBackend(
     tasks.toMap
   }
 
+  private def buildMesosLabels() : List[Label] = {
```

Comment:
Removed the space
The flatMap returns an Array, which doesn't work the with ScalaConverters, and the `addAlllabels` didn't take that Array.